docs/product-user-guide/src/main/asciidoc/guided-decision-tables-types-con.adoc

```diff
@@ -6,7 +6,7 @@ Two types of decision tables are supported in {PRODUCT}: Extended entry and Limi
 * *Extended entry:* An Extended Entry decision table is one for which the column definitions specify Pattern, Field, and Operator but not value.
 The values, or states, are themselves held in the body of the decision table.
 It is normal, but not essential, for the range of possible values to be restricted by limiting entry to values from a list.
-Business Central supports use of Java enumerations or decision table "optional value lists" to restrict value entry.
+{CENTRAL} supports use of Java enumerations or decision table "optional value lists" to restrict value entry.
```

Comment:
This sentence is not specific to "Extended Entry" but applies equally to "Limited Entry".

Furthermore there are three types of "discrete values: Java enumerations, "optional value lists" **and** "Workbench enumerations" (see [here](http://file.rdu.redhat.com/~sterobin/BXMSDOC-Vidya_BRMS/#enumerations_con) for example - IDK where it may live in the "real" documents).